app/src/main/java/de/westnordost/streetcomplete/quests/memorial_type/MemorialType.kt

```diff
@@ -0,0 +1,36 @@
+package de.westnordost.streetcomplete.quests.memorial_type
+
+import de.westnordost.streetcomplete.data.osm.osmquests.Tags
+
+sealed interface MemorialTypeAnswer
+
+enum class MemorialType(val osmValue: String) : MemorialTypeAnswer {
+    STATUE("statue"),
+    BUST("bust"),
+    PLAQUE("plaque"),
+    WAR_MEMORIAL("war_memorial"),
+    STONE("stone"),
+    OBELISK("obelisk"),
+}
+
+enum class Stele(val osmMaterialValue: String) : MemorialTypeAnswer {
+    WOODEN_STELE("wood"),
+    STONE_STELE("stone"),
+    KHACHKAR_STELE("stone"),
+}
+
+fun MemorialTypeAnswer.applyTo(tags: Tags) {
+    when (this) {
+        is MemorialType -> {
+            tags["memorial"] = this.osmValue
+        }
+        is Stele -> {
+            tags["memorial"] = "stele"
+            tags["material"] = this.osmMaterialValue
+        }
+    }
+
+    if (this == Stele.KHACHKAR_STELE) {
+        tags["stele"] = "khachkar"
+    }
```

Comment:
Maybe inlining the condition makes it more understandable:

```suggestion
        is Stele -> {
            tags["memorial"] = "stele"
            tags["material"] = this.osmMaterialValue

            if (this == Stele.KHACHKAR_STELE) {
                tags["stele"] = "khachkar"
            }
        }
    }
```

Reply:
Ah I see. Then I see no reason to include that in this quest. Unless there are stone steles, and khachkar are the only sub-type of stone steles that are defin(able).

Reply:
> khachkar are the only sub-type of stone steles that are defin(able)

right now - the only defined

I have no ideas for additional ones (except that gravestones are technically subtype of steles)

Reply:
> Could there not be hundreds of additional choices for different types of stone steles, different types of stones, different types of ... you get the idea ... if we allowed such differenciation on the top-level?

there could be - but right now khachkars are only known defined subtype

Reply:
I see. Well, I am trying to get to some sort of mutual decision and not "what I say goes".

From my point of view, the following reasons speak against it:
- ⚛️ Atomicy: A quest should ask the least possible (=taggable) detail
- 🪱 Can of worms: If such a specific sub-category of stone steles is included, what is the argument that can be used to not also include all kinds of further sub-categories (once theere is an established tagging for that). E.g. (mock/decorative) totem poles, cantabrian stelae, kurgan stelae, obelisks, stećak or those found in Copán and other places in central America (they are also very distinguishable from steles in Europe) etc.

The one reason that seems to speak for it is:
- the `stele=*` subtag is not exhaustive in the same way `service=*` for `highway=service` is not exhaustive. Hence, if it is not tagged now, it will probably never be taggable with StreetComplete

In my point of view, the reasons against it somewhat outweigh the reason for it. You don't agree?

Reply:
Yes, sadly it likely should be removed :(